src/snowflake/snowpark/modin/pandas/window.py

```diff
@@ -199,18 +199,6 @@ def count(self, numeric_only: bool = False):
         # TODO: SNOW-1063358: Modin upgrade - modin.pandas.window.Rolling
         return self._aggregate(method_name="count", numeric_only=numeric_only)
 
-    def sem(
```

Comment:
Why was this moved down?

Reply:
Moved this down just to match pandas order of aggregations in their documentation: https://pandas.pydata.org/docs/reference/window.html